keras/src/backend/openvino/numpy.py

```diff
@@ -1030,9 +1032,29 @@ def moveaxis(x, source, destination):
 
 
 def nan_to_num(x, nan=0.0, posinf=None, neginf=None):
-    raise NotImplementedError(
-        "`nan_to_num` is not supported with openvino backend"
-    )
+    x = get_ov_output(x)
+    dtype = x.get_element_type()
+    shape_x = ov_opset.shape_of(x).output(0)
+    nan_vector = ov_opset.broadcast(
+        ov_opset.constant(nan, dtype), shape_x
+    ).output(0)
```

Comment:
it can be just a scalar. select operation will broadcast it automatically. No need broadcast by yourself.

Reply:
the same comment for `pos_inf` and `neg_inf`:)

Reply:
done